### ISSUE TITLE
Changing permissions of Group per world as well as individual.

### ISSUE THREAD

**Issue by EvilPandaMan** [lucko/LuckPerms#2730]
Is is possible to update it with a group per world permission? For example, the vip group can use all their permissions in the world, world_nether but not in the world_the_end. That would remove the need to go through selecting each individual permission. 

A "add your worlds" menu that allows you to create a quick selector of worlds where permissions and groups apply, like a checkbox or something.

**Comment by Fefo6644** [comment]
You can already achieve per-world parentship and permissions with what we call "Contexts", they allow parent groups, permissions, prefixes, etc. to be applied when a certain set of conditions (the user's "contexts") are met. https://luckperms.net/wiki/Context :)

**Comment by EvilPandaMan** [comment]
No, that isn't exactly what I have suggested. Here, I made an example. Either by excluding what world that permission would exclude or the other way around. When you create a group, you assign a world to that group so everything within that group inherits that context. So I could create groups that only works in specific worlds.

![context](https://user-images.githubusercontent.com/22263166/99654288-0127c300-2a52-11eb-85b9-9469a6c555f5.png)
